### ISSUE TITLE
Indicator aiming at the floor

### ISSUE THREAD

**Issue by LunaCasey** [Simonrye/valorant-lineup-tool#1]
-

**Comment by Simonrye** [comment]
I'm sorry, I cannot access the video. Can you try to send the video again, or using a different method?

**Comment by LunaCasey** [comment]
> I'm sorry, I cannot access the video. Can you try to send the video again, or using a different method?

https://streamable.com/jxwbbz

Sorry, the old link expired.

**Comment by Simonrye** [comment]
Have you added your own Valorant sensitivity to settings.json? It is by default set to 0.4.

**Comment by snakeshka** [comment]
the same problem, the correct sensitivity is set in the settings (maybe this happens when you try to throw a lineup from the defense side)

**Comment by LunaCasey** [comment]
> Have you added your own Valorant sensitivity to settings.json? It is by default set to 0.4.

Yes, all the settings are correct. Mouse acceleration is off too. My sens is 0.35 and I changed it in the settings.json.

**Comment by Simonrye** [comment]
When you click the right arrow, does the distance in the console match the distance to the spike in-game?

Also, try to tab out of the Valorant window while keeping the mini-map and the spike visible. See the explanation video in the repo for reference.
Is there any obvious differences between my video and your screen?

**Comment by Simonrye** [comment]
> the same problem, the correct sensitivity is set in the settings (maybe this happens when you try to throw a lineup from the defense side)

I believe the issue is the overlay moving down too quickly. I think it might be caused by a mouse tracking issue, and/or a Valorant sensitivity discrepancy.
This script will not be affected by the side in which you decide to shoot the projectile.

**Comment by LunaCasey** [comment]
> When you click the right arrow, does the distance in the console match the distance to the spike in-game?

It varies. It's between 0.8 and 1.2 meters off.
 
> Also, try to tab out of the Valorant window while keeping the mini-map and the spike visible. See the explanation video in the repo for reference. Is there any obvious differences between my video and your screen?

The only thing I've noticed is that when I press the right arrow my DPI doesn't change, which could be the issue.

**Comment by Simonrye** [comment]
> The only thing I've noticed is that when I press the right arrow my DPI doesn't change, which could be the issue.

This definitely sounds like the issue. If the mouse doesn't slow down, it ruins the sync between game and mouse.

Inside the Python file, there is a change speed function.
Inside a new file, could you test the function to maybe find a fix?
Try running as administrator as well.

Also, what environment are you running?

**Comment by LunaCasey** [comment]
Well, never mind. I got permanently banned for trying to use it in custom games.

**Comment by Simonrye** [comment]
> Well, never mind. I got permanently banned for trying to use it in custom games.

I'm terribly sorry to hear. 
I have recently spent many hours testing the tool in customs, and many more for development so this is very shocking.
